code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -56,7 +56,15 @@
 
 /mob/living/carbon/human/movement_tally_multiplier()
 	. = ..()
-
+	if(locate(/obj/item/weapon/kitchen/utensil/knife/tactical) in held_items)
```

Comment:
Not to be too much of a party pooper, but a `locate` call in the middle of the main movement proc might cause some overhead and I'm pretty certain that a 1/1000 difference will be noticeable

Reply:
I asked around if there was a better thing to do than use an isType() check and I got told to do this by Exxion. Shrug.

Reply:
You didn't say you were doing it in the main moveloop